### DISCUSSION TITLE
Play mp4 video file in the learning path and the video size is double by default

### DISCUSSION THREAD

**Discussion by Ringoinmel** [chamilo/chamilo-lms#5195] [General]
When I just add mp4 video file in the learning path, there is no way I can adjust the video size and it is double the size and not clear.  The only solution I find is I have to insert it to a document first.  Anyone have ideas on how to modify the php files to fix it?  Thanks

**Comment by etree2024** [comment]
The document allows this to happen your other choice is change the Video size. Else we use SCORM tools and inbox them in SCORM but.

**Comment by Ringoinmel** [comment]
do you know which file control the video size?  I want to simply put the mp4 file and able to show it in the default normal size.  Thanks

**Reply by etree2024** [reply]
I use this app  handbrake https://handbrake.fr . Its Free. Good to have to manage videos parameters.

**Comment by ywarnier** [comment]
Chamilo 1.11 doesn't deal very well with direct video views in the learning path tool. Hopefully, this will be fixed in Chamilo 2.0 (not sure yet), but you could hack it a bit in main/lp/lp_view.php and main/template/default/learnpath/view.tpl... I'm not sure what you expect to do, though... Showing an MP4 video "as is" relies on the web browser capability to understand the video size and print it accordingly.
